optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -833,23 +836,6 @@ def best_booster(self) -> "lgb.Booster":
 
         return self.get_best_booster()
 
-    def _tune_params(
-        self,
-        target_param_names: List[str],
-        n_trials: int,
-        sampler: optuna.samplers.BaseSampler,
-        step_name: str,
-    ) -> _OptunaObjective:
-
-        objective = super(LightGBMTuner, self)._tune_params(
-            target_param_names, n_trials, sampler, step_name
-        )
-
-        if objective.best_booster_with_trial_number is not None:
-            self._best_booster_with_trial_number = objective.best_booster_with_trial_number
-            self._best_params.update(self.best_params)
```

Comment:
I noticed that `self._best_params` is not used by any part of the code. We can see two other lines (L387 and L401), so could you remove them too?

Reply:
Thank you for fixing them in https://github.com/optuna/optuna/pull/1702/commits/de499ce53148ab6e4bea7cb15348e37bed7397f7.